### ISSUE TITLE
Remove references to pylab from the script?

### ISSUE THREAD

**Issue by jypeter** [rougier/matplotlib-tutorial#13]
Salut,

I'm thinking you should maybe remove the (very) few references to pylab in your tutorial, as the offcial documentation now says "[The pylab API (disapproved)](https://matplotlib.org/api/index.html?highlight=pylab#the-pylab-api-disapproved)"

**Comment by rougier** [comment]
You're right, thanks. Now I need to find some time to do it... Or can you make a PR?
